client/src/assets/scss/overrides.scss

```diff
@@ -24,8 +24,8 @@
 
   .text-body-3 {
     font-family: inherit;
-    font-size: 1rem;
-    line-height: 1.375rem;
+    font-size: 14px;
+    line-height: 22px;
```

Comment:
1. we are trying to use rems for sizes whenever possible
2. is there any other text using this style that is now broken?

Reply:
Ya I know. Tracey specifically wants 14px for those texts. It is not used elsewhere

Reply:
14px = 0.875rem